src/java/org/apache/cassandra/tools/SSTableMetadataViewer.java

```diff
@@ -422,8 +422,6 @@ private void printSStableMetadata(File file, boolean scan) throws IOException
             field("ClusteringTypes", clusteringTypes.toString());
             field("StaticColumns", FBUtilities.toString(statics));
             field("RegularColumns", FBUtilities.toString(regulars));
-            if (stats != null)
```

Comment:
Remove `IsTransient` from SSTableMetadataViewer because sstables won't have a transient flag going forward and there is no need to look back for an experimental feature